packages/client/hmi-client/src/workflow/ops/dataset/tera-dataset-drilldown.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<tera-drilldown :title="node.displayName" @on-close-clicked="emit('close')">
+		<section tabName="Description">
+			<tera-drilldown-section :is-loading="fetchingDataset">
+				<div>
+					<tera-dataset-description :dataset="dataset" :raw-content="rawContent" />
+				</div>
```

Comment:
remove uneeded div

Reply:
i needed to move this div to the description, component.  It was hiding content without it

Reply:
hmm it needed a proper parent element anyways I guess